tests/manifolds/test_symmetric_positive_definite.py

```diff
@@ -66,6 +75,25 @@ def test_dist(self):
             manifold.dist(x, y), manifold.dist(axa, aya)
         )
 
+        def geodesic(point_a, point_b, alpha):
+            if alpha < 0 or 1 < alpha:
+                raise ValueError("Exponent must be in [0,1]")
+            c = np.linalg.cholesky(point_a)
+            c_inv = np.linalg.inv(c)
+            logm = multilogm(
+                multiprod(multiprod(c_inv, point_b), multitransp(c_inv)),
+                positive_definite=True,
+            )
+            powm = multiexpm(alpha * logm, symmetric=False)
+            return multiprod(multiprod(c, powm), multitransp(c))
+
+        # Test proportionality (see equation (6.12) in [Bha2007]).
+        alpha = np.random.rand()
```

Comment:
I gave the wrong suggestion last time around :sweat_smile: 

```suggestion
        alpha = np.random.uniform()
```